### PR TITLE
C_CPP : added  `Looping Control Structure.md` in C/C++

### LINKED ISSUE #863: C_CPP : Looping Control Structure
<hr>

## Description 📜

I want to contribute on Looping Control Structure in C using documentation.

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by thepranaygupta**
<hr>

## Description 📜

added documentation of Looping Control Structure in C/C++
Fixes #863

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

File: C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<div align="center">
```
Comment (kashishahuja2002) [inline C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md:1]: Don't use html in md file, unless necessary.

File: C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+<h1>Looping Control Structure in C/C++</h1>
+</div>
+
+## Loop/Iteration: The repeated execution of a statement or group of statements until a given condition is false or dissatisfied.
+
+## Types of Loops in C/C++
```
Comment (kashishahuja2002) [inline C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md:7]: Also include nested loops.

File: C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+1. **for loop** (Counter loop/single statement loop)
+2. **while loop** (Entry-controlled/Pre-conditional)
+3. **do while loop** (Entry-controlled/Post-conditional)
```
Comment (kashishahuja2002) [inline C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md:11]: Do-while loop is exit controlled. Proof-read your content before submitting.

File: C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+4. **goto loop** (Still exists but not in practice)
```
Comment (kashishahuja2002) [inline C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md:12]: Goto loop is still useful, don't write "not in practice".

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add content for nested and goto loop.

File: C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md (hunk 3 above)
Comment (thepranaygupta) [reply C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md:11]: sorry, I'll correct this.

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@thepranaygupta Please add comments to your code or  a little explanation after each code snippet.

**Pull request closed without merging**